### DISCUSSION TITLE
New Project Review

### DISCUSSION THREAD

**Discussion by vkdang** [community/community#37930] [Other Feature Feedback, Questions, & Ideas]
Hey all! New to the programming and dev world and working on a new project with a small startup. Anyone willing to give some feedback on my repository (and also chuck a star if possible :D). Would be much appreciated!
https://search-the-way-you-th.ink/3SRl6iC

**Comment by ossanna16** [comment]
Hi @vkdang, we’re glad you’re here 😀
Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
